### ISSUE TITLE
Insufficient Color Contrast for "Edit," "Take Photo," "Upload Image," and "Logout From Logged In Location" Buttons on Profile Screen

### ISSUE THREAD

**Issue by anirbanpaulcom** [RocketChat/Rocket.Chat.ReactNative#5547]
### Description:

The "Edit," "Take Photo," "Upload Image," and "Logout from Logged In Location" buttons on the profile screen exhibit insufficient color contrast against the gray background.

### Environment Information:

Rocket.Chat Server Version: 6.5.0
Rocket.Chat App Version: 4.47.0
Device Name: Redmi note 5 pro
OS Version: Android

### Steps to reproduce:

1. Go to the profile screen.
2. Observe the "Edit," "Take Photo," "Upload Image," and "Logout from Logged In Location" buttons Color.

### Expected behavior:
![WhatsApp Image 2024-02-07 at 7 58 03 AM](https://github.com/RocketChat/Rocket.Chat.ReactNative/assets/130109852/f29d94c9-d498-45ce-8dd0-0113b3e2c0ef)
![WhatsApp Image 2024-02-07 at 8 07 33 AM (1)](https://github.com/RocketChat/Rocket.Chat.ReactNative/assets/130109852/389f9b8e-cc64-4c52-852a-a73253ead27b)

![WhatsApp Image 2024-02-07 at 8 07 33 AM](https://github.com/RocketChat/Rocket.Chat.ReactNative/assets/130109852/b53aca8b-191c-4377-9a77-47cf36b00456)

### Actual behavior:
![WhatsApp Image 2024-02-07 at 9 22 40 AM](https://github.com/RocketChat/Rocket.Chat.ReactNative/assets/130109852/b0cfa31f-649d-4a62-b5a6-c8dc352bf964)
![WhatsApp Image 2024-02-07 at 9 02 34 AM](https://github.com/RocketChat/Rocket.Chat.ReactNative/assets/130109852/61914229-1e8d-4264-aee3-ded67c3bae03)

**Comment by anirbanpaulcom** [comment]
Fix :  #5548

**Comment by GleidsonDaniel** [comment]
> Fix : #5548

This type of fix doesn't work like that, this is a decision that the design team makes.
It's not something that came out of our heads.

I'll report back to the design team and see what they think.

**Comment by anirbanpaulcom** [comment]
> > Fix : #5548
> 
> This type of fix doesn't work like that, this is a decision that the design team makes. It's not something that came out of our heads.
> 
> I'll report back to the design team and see what they think.

thank you.
